scripts/gen_help_html.lua

```diff
@@ -512,6 +532,15 @@ local function visit_node(root, level, lang_tree, headings, opt, stats)
     end
     local el = in_heading and 'span' or 'code'
     local s = ('%s<a name="%s"></a><%s class="%s">%s</%s>'):format(ws(), url_encode(tagname), el, cssclass, trimmed, el)
+
+    -- Vim tabs take into account the two concealed '*' characters even
+    -- though they are invisible. If we happen to have a 6/7-length tag, we
+    -- need to inject two empty spaces to match Vim's behavior.
+    local next_node_text = node_text(root:next_sibling())
+    if string.sub(next_node_text,1,1) == "\t" then
+        s = s .. '  '
+    end
```

Comment:
Should be easy to extract this logic to a function (suggested name: `fix_tab_after_conceal()`), then we can avoid duplicating this everywhere.

Nit: 2-space indent :)

Reply:
Just run `stylua scripts/gen_help_html.lua` from the repo root ;)

Reply:
Ok I refactored the code to the function and fixed the indent.

I didn't run stylua though because it would fix up a bunch of other parts of the code :)